packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -59,12 +62,16 @@ const checkShouldUseMobileFeedLayout = (
 
 export const useFeedLayout = ({
   feedRelated = true,
+  originFeedName,
 }: UseFeedLayoutProps = {}): UseFeedLayoutReturn => {
   const isLaptop = useViewSize(ViewSize.Laptop);
   const { feedName } = useActiveFeedNameContext();
-
+  const evaluateFeedName =
+    originFeedName === OtherFeedPage.SourceMostUpvoted
+      ? originFeedName
```

Comment:
Hmm, how we were using this before is that when we would pass the feedname it would override the feedName from context, no matter what feed it was.

Personally think it makes more sense like that, any chance we can pass this prop only when feed is SourceMostUpvoted?

Reply:
This way and also from the changes above looks like we are passing this prop almost everywhere, which gives also the impression this is a required prop. 

Since we also have a route for this from what I see below `sources[source]/most-upvoted` why can't we evaluate like we do for others?

Reply:
Ah yes can do like that indeed 👍

Reply:
Maybe it could be a combination of `sources[source]/most-upvoted` + any additional prop we can take for the card ones which are not going to change to new layout

Reply:
Btw if this way is staying, more additional feednames need to be added on other places like `CardCover` ex.
<img width="822" alt="Screenshot 2024-04-19 at 3 48 01 PM" src="https://github.com/dailydotdev/apps/assets/36197304/2e50387b-2aa1-43d4-9957-75a990fc7e58">
Causes this shift in cover